tensorflow/python/data/experimental/kernel_tests/map_and_batch_test.py

```diff
@@ -22,6 +22,7 @@
 from absl.testing import parameterized
 import numpy as np
 
+from tensorflow.python.data.experimental.kernel_tests.serialization import dataset_serialization_test_base
```

Comment:
let's move the dataset_serialization_test_base to `tensorflow.python.data.kernel_tests` and rename it to `checkpoint_test_base.py`

Reply:
addressed in https://github.com/tensorflow/tensorflow/pull/47318